pkg/serdes.go

```diff
@@ -37,83 +37,112 @@ func readBodyBuffered(resp *http.Response) ([]byte, error) {
 }
 
 func imageFromBuffer(buffer []byte, transform *ARAwareTransform, aspectRatio float64, encodeImage bool, isMask bool) (*ImagePayload, float64, error) {
-	// Decode the image payload using vips
-	img, err := vips.NewImageFromBuffer(buffer)
+	// Decode the image payload using vips, using bulletproof settings
+	importParams := vips.NewImportParams()
```

Comment:
explicit and visible load settings, didn't make anything more stable but still a good thing probably